nixos/tests/qemu-vm-external-disk-image.nix

```diff
@@ -57,6 +57,10 @@ in
       # with virtualisation.fileSystems.
       virtualisation.fileSystems = lib.mkForce { };
 
+      # swith.enable can be set instead but
+      # this acts as a test for switch-to-configuration-ng with qemu-vm
+      system.switch.enableNg = true;
```

Comment:
I see https://github.com/NixOS/nixpkgs/blob/a7485af7d504540d9c82ef3e9ac1da1c8f75280a/nixos/modules/system/activation/switchable-system.nix#L20 `system.switch.enable` should be true by default, I am not sure why this error occured